x-pack/plugins/apm/server/lib/services/get_service_metadata_icons.ts

```diff
@@ -43,6 +46,9 @@ export async function getServiceMetadataIcons({
     { term: { [SERVICE_NAME]: serviceName } },
     { range: rangeFilter(start, end) },
     ...setup.esFilter,
+    ...getDocumentTypeFilterForAggregatedTransactions(
```

Comment:
Can you add a comment to the definition of `getDocumentTypeFilterForAggregatedTransactions` explaining when this should be used?

Reply:
@cauemarcondes on what basis did you decide to only query transaction data (whether raw events or metrics)?

Reply:
Btw, how does the fallback to `containerType` work? Why do we fall back to `Docker`?

Reply:
Also, why are we adding `setup.esFilter`? (apologies for all the questions 🙂)

Reply:
> on what basis did you decide to only query transaction data

I just saw where the data was and used it. If you think we should use all indices, or a specific one just let me know.

Reply:
> Btw, how does the fallback to containerType work? Why do we fall back to Docker?

@alex-fedotyev agreed to only show kibana, docker, or nothing. https://github.com/elastic/kibana/pull/84760

Reply:
> Also, why are we adding setup.esFilter? (apologies for all the questions 🙂)

np, that's indeed something that the query **should not** use. I'm going to remove it.